### PR TITLE
Fix purchase button

### PR DESCRIPTION
**Pull request by eugene-eeo**

### CODE REVIEW

File: src/javascript/binary/websocket_pages/trade/event.js [hunk 1]
```diff
@@ -342,6 +342,7 @@ var TradingEvents = (function () {
          */
         // using function expression form here as it used inside for loop
         var purchaseContractEvent = function () {
+            console.log('clicked!');
```
Comment (negar-binary) [inline src/javascript/binary/websocket_pages/trade/event.js]: did you mean to leave this in?

Comment (eugene-eeo) [inline src/javascript/binary/websocket_pages/trade/event.js]: yes, just for QA to verify that it works cross browser. after QA phase I will remove it.

File: src/javascript/binary/websocket_pages/trade/event.js [hunk 2]
```diff
@@ -340,8 +340,9 @@ var TradingEvents = (function () {
         /*
          * attach event to purchase buttons to buy the current contract
          */
-        // using function expression form here as it used inside for loop
-        var purchaseContractEvent = function () {
+        $('.purchase_button').on('click dblclick', function () {
+            if (isVisible(document.getElementById('confirmation_message_container'))) return;
+            console.log('clicked!');
```
Comment (negar-binary) [inline src/javascript/binary/websocket_pages/trade/event.js]: @eugene-eeo :)

Comment (eugene-eeo) [inline src/javascript/binary/websocket_pages/trade/event.js]: oops. can you remember to remove this in one of your PRs? 😅

Comment (negar-binary) [inline src/javascript/binary/websocket_pages/trade/event.js]: @eugene-eeo sure :)

**Pull request merged**